src/views/splash.html

```diff
@@ -0,0 +1,134 @@
+                                    Social
+                                </h4>
+                                <div>
+                                    <a href="https://www.facebook.com/hackcambridge">
+                                        <img id="splash-facebook-logo" src="{{ asset('images/hc-2018/facebook-logo.svg') }}" alt="Facebook logo" draggable="false"/>
+                                    </a>
+                                    <a href="https://twitter.com/@hack_cambridge">
+                                        <img id="splash-twitter-logo" src="{{ asset('images/hc-2018/twitter-logo.svg') }}" alt="Twitter logo" draggable="false"/>
+                                    </a>
+                                    <a href="https://medium.com/@HackCambridge">
+                                        <img id="splash-medium-logo" src="{{ asset('images/hc-2018/medium-logo.svg') }}" alt="Medium logo" draggable="false"/>
+                                    </a>
+                                    <a href="https://www.instagram.com/hack_cambridge/">
+                                        <img id="splash-instagram-logo" src="{{ asset('images/hc-2018/instagram-logo.svg') }}" alt="Instagram logo" draggable="false"/>
+                                    </a>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </section>
+            <section id="splash-signup" class="red">
+                <div class="container width">
+                    <div class="one half unit row">
+                        <div class="six unit h"></div>
+                        <div class="seven unit h">
+                            <div class="container">
+                                <div class="one unit row">
+                                    <div class="five unit h">
+                                        <h5>
+                                            Sign up now!<br><span class="secondary">We’ll be in touch soon.</span>
+                                        </h5>
+                                    </div>
+                                </div>
+                                <div class="half unit row">
+                                    <div class="five unit h">
+                                        <form id="splash-signup-form" class="half unit row" method="post" action="some-action">
+                                            <div class="four small unit h">
+                                                <input type="email" name="signup-email" required placeholder="Enter your email address." />
+                                            </div>
+                                            <div class="small unit h"></div>
+                                            <div class="one unit h">
+                                                <input type="submit" name="submit" value="Submit." />
```

Comment:
It's a small thing, but I think it would make more sense if the dialog box didn't have a full stop, since the action isn't a sentence.  What's your view?

Reply:
Not a fan of http://www.three.co.uk? 😉
My idea with buttons is that `→` is used to indicate a continuing action, `.` a completing action (like submit, cancel, finish…), and `!` a dangerous action (like delete)

Reply:
I think it's definitely unconventional.  The `!` is arguably useful but for `→` and `.` I don't really see the point of adding the complexity — personally I'd follow convention to make flow familiar and easy.

Reply:
How about `!` and `→` but no`.`? I think the meaning of `→` is pretty obvious—there are many websites with buttons like `Next →`. And besides inconsistent use of `↗︎` for external links prevails seemingly without causing confusion.

Reply:
I would agree that the meaning is obvious, my point was more that `→` can be redundant.